components/Slider/examples/SliderCompareExampleViewController.m

```diff
@@ -17,60 +17,78 @@
 #import <UIKit/UIKit.h>
 
 #import "MaterialColorScheme.h"
+#import "MaterialTypographyScheme.h"
 #import "MaterialMath.h"
 #import "MaterialSlider.h"
 #import "MaterialSlider+ColorThemer.h"
 
 @interface SliderCompareExampleViewController : UIViewController
 @property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
+@property(nonatomic, strong) MDCTypographyScheme *typographyScheme;
 @end
 
-@implementation SliderCompareExampleViewController
+@implementation SliderCompareExampleViewController {
+  MDCSlider *_slider;
```

Comment:
Nit/Consistency: Let's stick with Properties instead of adding ivars as well.

Reply:
Done